src/huggingface_hub/file_download.py

```diff
@@ -671,6 +672,13 @@ def cached_download(
             " 'hf_hub_download'",
             FutureWarning,
         )
+    if resume_download:
+        warnings.warn(
+            "`resume_download` is deprecated and will be removed in version 1.0.0. "
+            "Download always resume when possible. "
```

Comment:
```suggestion
            "Downloads always resume when possible. "
```

Reply:
How about passing the default of `resume_download` to `None` and having the following warning additionally:

```py
if not resume_download and resume_download is not None:
    warnings.warn("resume_download is deprecated and will be removed in 1.0.0. In order to force a download, please use force_download=True")
```

Reply:
(if a user was passing this arg with `False` they wouldn't be aware of that change/the recommended argument)

Reply:
Good idea yes! I've updated this in https://github.com/huggingface/huggingface_hub/pull/2223/commits/92106486701ef8175920b0bade7bf84f5e3c010e